front-end/src/app/components/home/Home.scss

```diff
@@ -0,0 +1,42 @@
+@import "../../../assets/scss/colors";
+
+/* Rules for sizing the icon. */
+.material-icons.md-18 { font-size: 18px; }
+.material-icons.md-24 { font-size: 24px; }
+.material-icons.md-36 { font-size: 36px; }
+.material-icons.md-48 { font-size: 48px; }
+.material-icons.md-96 { font-size: 96px; }
+
+/* Button container */
+.home_button {
+    margin: auto;
+    max-width: 720px;
+    align-items: center;
+    justify-content: center;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+}
+
+/* purple button */
+.home_button .purple-button {
```

Comment:
You should take advantage of SCSS on this one ; here you're just writing plain CSS.

You can do something like:
```SCSS
.home_button {
    margin: auto;
    [...]
  .purple-button {
    [...]
  }
}
```

See [nesting in doc](https://sass-lang.com/guide#nesting)